src/docs/product/cli/installation.mdx

```diff
@@ -100,3 +102,7 @@ docker run --rm -v $(pwd):/work getsentry/sentry-cli --help
 ## Updating and Uninstalling
 
 You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the `sentry-cli` binary. These commands may be unavailable in certain situations, generally when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
+
+## Build Checksums
+
+<CliChecksumTable />
```

Comment:
It's likely not everyone knows what these are for or what to do with them. It might be helpful to have just one or two lines above the table explaining what people are even looking at.

Reply:
I agree, I think this would be a good addition.